### PR TITLE
docs - ALTER TABLE SET DISTRIBUTED for external tables

### PR DESCRIPTION
**Pull request by lisakowen**
docs for https://github.com/greenplum-db/gpdb/pull/14199.

greenplum 7 supports changing the distribution policy of an external table via ALTER TABLE.

### CODE REVIEW

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md [hunk 1]
```diff
@@ -23,8 +23,9 @@ ALTER TABLE [IF EXISTS] <name>
 ALTER TABLE ALL IN TABLESPACE <name> [ OWNED BY <role_name> [, ... ] ]
     SET TABLESPACE <new_tablespace> [ NOWAIT ]
 
-ALTER TABLE [IF EXISTS] [ONLY] <name>
-   | DISTRIBUTED BY ({<column_name> [<opclass>]} [, ... ] )
+ALTER TABLE [IF EXISTS] [ONLY] <name> SET
+   [ WITH (reorganize=true|false) ]
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:27]: Capitalize "reorganize" here.

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md [hunk 2]
```diff
@@ -316,14 +317,16 @@ value
...
-:   Changing to or from a replicated distribution policy causes the table data to be redistributed.
+:   Changing to or from a replicated distribution policy always causes the table data to be redistributed.
+:   Greenplum Database permits changing the distribution policy of a writable external table to `DISTRIBUTED BY` or `DISTRIBUTED RANDOMLY` only.
 
 analyze_hll_non_part_table=true|false
 :   Use `analyze_hll_non_part_table=true` to force collection of HLL statistics even if the table is not part of a partitioned table. The default is `false`.
 
 reorganize=true\|false
 :   Use `reorganize=true` when the hash distribution policy has not changed or when you have changed from a hash to a random distribution, and you want to redistribute the data anyway.
+:   If you are setting the distribution policy, you must specify the `WITH (reorganize=<value>)` clause before the `DISTRIBUTED ...` clause.
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:328]: Capitalize again?

**Review by dyozie — APPROVED** [review]

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md (hunk 1 above)
Comment (lisakowen) [reply gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:27]: ok, so i originally had REORGANIZE, but then i saw that it was lowercase in the storage_parameter list, and also in the options discussion.

Comment (lisakowen) [reply gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:27]: i'll investigate further.

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md [hunk 3]
```diff
@@ -316,14 +317,16 @@ value
 
 DISTRIBUTED BY \(\{column\_name \[opclass\]\}\) \| DISTRIBUTED RANDOMLY \| DISTRIBUTED REPLICATED
 :   Specifies the distribution policy for a table. Changing a hash distribution policy causes the table data to be physically redistributed, which can be resource intensive. If you declare the same hash distribution policy or change from hash to random distribution, data will not be redistributed unless you declare `SET WITH (reorganize=true)`.
-
-:   Changing to or from a replicated distribution policy causes the table data to be redistributed.
+:   Changing to or from a replicated distribution policy always causes the table data to be redistributed.
+:   Greenplum Database permits changing the distribution policy of a writable external table to `DISTRIBUTED BY` or `DISTRIBUTED RANDOMLY` only.
```
Comment (huansong) [inline gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:321]: Though not tested by https://github.com/greenplum-db/gpdb/pull/14199, but we can alter it to `DISTRIBUTED REPLICATED` too. Again it's just a metadata change, not actual distribution.

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md (hunk 1 above)
Comment (huansong) [reply gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:27]: +1 lower-case to be consistent within the same page.

**Review by huansong — APPROVED** [review]
LGTM, one minor change needed.

**Pull request merged**